docs/src/main/asciidoc/cdi-reference.adoc

```diff
@@ -1069,6 +1069,47 @@ public class NoopAsyncObserverExceptionHandler implements AsyncObserverException
 }
 ----
 
+[[reactive_pitfalls]]
+== Pitfalls with Reactive Programming
+
+CDI is a purely synchronous framework.
+Its notion of asynchrony is very limited and based solely on thread pools and thread offloading.
```

Comment:
is that the case for async event/observers ?

Reply:
Yes. That's the only place in CDI where asynchrony is mentioned, and it's based on thread offloading.